manifests/charts/install-OpenShift.md

```diff
@@ -25,7 +25,7 @@ helm install istio-cni -n kube-system manifests/charts/istio-cni --set cni.cniBi
 3) `istio-control/istio-discovery` chart installs a revision of istiod.
 
 ```console
- helm install -n istio-system istio-17 manifests/charts/istio-control/istio-discovery --set istio_cni.enabled=true --set global.jwtPolicy=first-party-jwt --set sidecarInjectorWebhook.injectedAnnotations."k8s\.v1\.cni\.cncf\.io/networks"="istio-cni"
+ helm install -n istio-system istio-17 manifests/charts/istio-control/istio-discovery --set istio_cni.enabled=true --set istio_cni.chained=false --set global.jwtPolicy=first-party-jwt 
```

Comment:
do we need `global.jwtPolicy=first-party-jwt`? hopefully not? Its not in OpenShift IOP profile

Reply:
Its also on a few other lines btw, if we do end up dropping it

Reply:
I don't think we do. I don't have my openshift cluster up and running anymore. I'll try it the next time I do.